docs/schema/constants/gradient-fill-type.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "allOf": [
+        {
+            "const": "gf",
```

Comment:
Why are these needed as separate constants? I think it adds complexity to the schema unnecessarily

Reply:
so we don't need to maintain the same values at many places. For example, `gradient-fill-type` has been used in two files:
1. https://github.com/LottieFiles/lottie-docs/pull/10/files#diff-d16a802ae7799a373c918b7c1ef3060dcc26d7bc7056fed9ce61f839d5b0a4f7R16
2. https://github.com/LottieFiles/lottie-docs/pull/10/files#diff-07b1cf6f3056eccc7be2ada3b3d6157b246ba8baab03587f53f9c9b7e28015f8R41

Reply:
dunno, seems like a lot of overhead

Reply:
but shouldn't we create this overhead if a value/property has been used in more than one place?